linera-views/src/reentrant_collection_view.rs

```diff
@@ -745,14 +776,15 @@ where
                     &values[i_key * num_init_keys..(i_key + 1) * num_init_keys],
```

Comment:
Here, we could save 1 multiplication if we wanted :)

Reply:
I think I can refactor this to not need the manual index calculations. What do you think of [this](https://github.com/jvff/linera-protocol/commit/d80fea02a099f0b1728afb57d54e9d3cad8bf8ae)? I can open a PR with that change after this one is merged :+1:

Reply:
@jvff Nice!